app/controllers/votes_controller.rb

```diff
@@ -0,0 +1,21 @@
+class VotesController < ApplicationController
+  def create 
+    @vote = Vote.new(
+      user_id: 1,
+      work_id: params[:work_id]
```

Comment:
Instead of setting `user_id` to 1 here, you should set it to `session[:user_id]`.

As is, every vote always comes from user 1 instead of whoever is logged in.